src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -101,43 +108,65 @@ internal struct NameDeclarationInfo
 
             private static bool IsPossibleVariableOrLocalMethodDeclaration(
                 SyntaxToken token, SemanticModel semanticModel, int position,
-                CancellationToken cancellationToken, out NameDeclarationInfo result)
+                CancellationToken cancellationToken, out ImmutableArray<NameDeclarationInfo> result)
```

Comment:
Seems like a bunch of these were updated, but didn't need to be.  Not sure the consistency is worth it in this case.  Can't a caller do the wrapping for a bunch of these?